src/libsync/discovery.cpp

```diff
@@ -800,8 +800,14 @@ void ProcessDirectoryJob::processFileAnalyzeLocalInfo(
                || localEntry.isDirectory || localEntry.isVirtualFile);
 
     if (isMove) {
-        //  The old file must have been deleted.
-        isMove = !QFile::exists(_discoveryData->_localDir + base._path);
+        // The old file must have been deleted.
+        isMove = !QFile::exists(_discoveryData->_localDir + base._path)
+#ifdef Q_OS_WIN
```

Comment:
I believe we should use Utility::fsCasePreserving,  not?

Reply:
Yes

Reply:
Updated.